bin/intl_sorting.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Maybe call it sort_intl.dart so it's more like an action?

Reply:
Rename file